_config.yml

```diff
@@ -21,7 +21,8 @@ nonce: a671871a966400ff97d0293ed2e1568c
 state: abcdefghijklmnopabcdefghijklmnop
 
 copy_to_destination:
-  - node_modules/identity-style-guide/dist/assets
+  - source: node_modules/uswds/dist/{js,fonts,img}
```

Comment:
The braces get expanded by `Dir.glob` in the plug-in right?

Reply:
> The braces get expanded by `Dir.glob` in the plug-in right?

Correct. There'd also be clear breakage if it wasn't working (wrong fonts, accordion content open-always and untoggle-able, missing `+` button images on accordion toggles).